pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -922,3 +923,23 @@ def update(self, request, *args, **kwargs):
             return self._get(request, *args, **kwargs)
 
     return GeneratedRedirectView.as_view(actions, url=url)
+
+
+class ClientConfigurationView(views.APIView):
+    """Return configurations for the ansible-galaxy client."""
+
+    DEFAULT_ACCESS_POLICY = _PERMISSIVE_ACCESS_POLICY
+
+    @extend_schema(responses=ClientConfigurationSerializer)
+    def get(self, request, *args, **kwargs):
+        """Get the client configs."""
+
+        data = ClientConfigurationSerializer(
+            {
+                "default_distribution_path": self.kwargs.get(
+                    "path", settings.ANSIBLE_DEFAULT_DISTRIBUTION_PATH
```

Comment:
I think, we need to add the path as a query string to the schema.

Reply:
The path being part of the url pattern was not completely obvious to me at this point. Sorry.